Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -63,6 +63,36 @@ select new
             return Json(await equipment.ToListAsync());
         }
 
+        [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Equipment>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> SearchAttributes(string q)
+        {
+            var attributes = from ea in _context.EquipmentAttributes
+                    .Where(a => a.Equipment.Team.Slug == Team && a.Equipment.Active &&
+                                (EF.Functions.Like(a.Value, q.EfStartsWith())))
+                    .AsNoTracking()
+                select new
+                {
+                    ea.Equipment.Id
+                };
+            var ids = await attributes.Distinct().Select(a => a.Id).ToArrayAsync();

```

Comment:
@jSylvestre Why are you doing a select here when you already projected in the previous `attributes` query?  Doesn't seem like it would do anything

Reply:
@srkirkland If I don't it gives me a compile error:

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1929	'<anonymous type: int Id>[]' does not contain a definition for 'Contains' and the best extension method overload 'MemoryExtensions.Contains<int>(ReadOnlySpan<int>, int)' requires a receiver of type 'ReadOnlySpan<int>'	Keas.Mvc	D:\Work\GitProjects\Peaks\Keas.Mvc\Controllers\Api\EquipmentController.cs	82	Active

Reply:
Or I can just do the select correctly....